There are Checkstyle errors. Could you please look into them?

```
[29](https://github.com/exercism/java/actions/runs/24846500338/job/72808756337?pr=3108#step:4:730)
Error: eckstyle] [ERROR] /home/runner/work/java/java/exercises/practice/connect/src/test/java/ConnectTest.java:166:17: 'array initialization' child has incorrect indentation level 16, expected level should be 12. [Indentation]

Error: eckstyle] [ERROR] /home/runner/work/java/java/exercises/practice/connect/src/test/java/ConnectTest.java:167:17: 'array initialization' child has incorrect indentation level 16, expected level should be 12. [Indentation]
Error: eckstyle] [ERROR] /home/runner/work/java/java/exercises/practice/connect/src/test/java/ConnectTest.java:168:17: 'array initialization' child has incorrect indentation level 16, expected level should be 12. [Indentation]
Error: eckstyle] [ERROR] /home/runner/work/java/java/exercises/practice/connect/src/test/java/ConnectTest.java:169:17: 'array initialization' child has incorrect indentation level 16, expected level should be 12. [Indentation]
Error: eckstyle] [ERROR] /home/runner/work/java/java/exercises/practice/connect/src/test/java/ConnectTest.java:188:17: 'array initialization' child has incorrect indentation level 16, expected level should be 12. [Indentation]
Error: eckstyle] [ERROR] /home/runner/work/java/java/exercises/practice/connect/src/test/java/ConnectTest.java:189:17: 'array initialization' child has incorrect indentation level 16, expected level should be 12. [Indentation]
Error: eckstyle] [ERROR] /home/runner/work/java/java/exercises/practice/connect/src/test/java/ConnectTest.java:190:17: 'array initialization' child has incorrect indentation level 16, expected level should be 12. [Indentation]
Error: eckstyle] [ERROR] /home/runner/work/java/java/exercises/practice/connect/src/test/java/ConnectTest.java:191:17: 'array initialization' child has incorrect indentation level 16, expected level should be 12. [Indentation]
```

--- a/exercises/practice/connect/src/test/java/ConnectTest.java
+++ b/exercises/practice/connect/src/test/java/ConnectTest.java
@@ -163,10 +163,10 @@ public class ConnectTest {
 
         //GIVEN
         var board = new String[]{
-                ". . X .",
-                " X X . .",
-                "  . X X X",
-                "   O O O O"
+            ". . X .",
+            " X X . .",
+            "  . X X X",
+            "   O O O O"
         };
         Connect cut = new Connect(board);
 
@@ -185,10 +185,10 @@ public class ConnectTest {
 
         //GIVEN
         var board = new String[]{
-                ". . X X",
-                " X X . .",
-                "  . X X .",
-                "   O O O O"
+            ". . X X",
+            " X X . .",
+            "  . X X .",
+            "   O O O O"
         };
         Connect cut = new Connect(board);
 

